e2e/config/config.go

```diff
@@ -54,21 +56,36 @@ func (c *Configuration) BlobStorageAccountURL() string {
 }
 
 func (c *Configuration) GalleryResourceID() string {
-	return fmt.Sprintf("/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Compute/galleries/%s", c.GallerySubscriptionID, c.GalleryResourceGroupName, c.GalleryName)
+	return fmt.Sprintf("/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Compute/galleries/%s", c.GallerySubscriptionID, c.GalleryResourceGroup, c.GalleryName)
```

Comment:
same, ResourceGroupName and ResourceGroupID are often used in the SDK. I tend to prefer the explicit naming.